libs/deps/merch-card.js

```diff
@@ -797,6 +797,8 @@ merch-card[variant="mini-compare-chart"] [is="inline-price"] {
     display: inline-block;
     min-height: 30px;
     min-width: 1px;
+    max-width: -moz-available;
+    max-width: -webkit-fill-available;
```

Comment:
Since this is text, wouldn't a `word-break: break-all` or something similar achieve the same result? Had a very quick look and the two solutions seem to yield the same results.